platform/openide.util/test/unit/src/org/openide/util/PairTest.java

```diff
@@ -19,39 +19,90 @@
 package org.openide.util;
 
 import org.netbeans.junit.NbTestCase;
-import org.openide.util.Pair;
 
 /**
  *
- * @author Tomas Zezula
+ * @author Tomas Zezula, Lukasz Bownik
  */
 public class PairTest extends NbTestCase {
 
-    public PairTest(String name) {
+    //--------------------------------------------------------------------------
+    public PairTest(final String name) {
+
         super(name);
     }
 
+    //-------------------------------------------------------------------------- 
+    public void test_pairEquality() {
```

Comment:
Please use camelCase and not snakeCase

Reply:
fixed in https://github.com/apache/netbeans/pull/5115/commits/68bc78dc9355a667f7d8208c9e0f39bf53c18013